test/peer-store/peer-store.spec.js

```diff
@@ -161,19 +161,22 @@ describe('peer-store', () => {
     expect(peerStore.peers.size).to.equal(0)
   })
 
-  it('should be able to remove a peer from store through its b58str id', async () => {
-    const [peerInfo] = await peerUtils.createPeerInfo()
+  it('should be able to get the multiaddrs for a peer', async () => {
```

Comment:
This test was improperly named and was just a copy+paste from the previous test name.